network.go

```diff
@@ -284,12 +305,26 @@ func (n *network) CopyTo(o datastore.KVObject) error {
 	dstN.dbExists = n.dbExists
 	dstN.drvOnce = n.drvOnce
 
+	for _, v4conf := range n.ipamV4Config {
+		dstV4Conf := &IpamConf{}
+		v4conf.CopyTo(dstV4Conf)
+		dstN.ipamV4Config = append(dstN.ipamV4Config, dstV4Conf)
+	}
+
 	for _, v4info := range n.ipamV4Info {
 		dstV4Info := &IpamInfo{}
 		v4info.CopyTo(dstV4Info)
 		dstN.ipamV4Info = append(dstN.ipamV4Info, dstV4Info)
 	}
 
+	if n.ipamV6Config != nil {
```

Comment:
Here we do, in datastore object should be the same as in cache. A nil list of v6 ipam config has to remain nil, not an empty list.